packages/react-scripts/scripts/init.js

```diff
@@ -123,7 +137,13 @@ module.exports = function(
     console.log(`Installing react and react-dom using ${command}...`);
     console.log();
 
-    const proc = spawn.sync(command, args, { stdio: 'inherit' });
+    const proc = spawn.sync(
+      command,
+      args.concat(require('./utils/dependencies')),
```

Comment:
Generation of the `dependencies` section inside the `package.json` for the template now based on the `./utils/dependencies.js` content.